pkg/cmd/gist/create/create_test.go

```diff
@@ -281,10 +299,15 @@ func Test_createRun(t *testing.T) {
 	}
 	for _, tt := range tests {
 		reg := &httpmock.Registry{}
-		reg.Register(httpmock.REST("POST", "gists"),
-			httpmock.JSONResponse(struct {
-				Html_url string
-			}{"https://gist.github.com/aa5a315d61ae9438b18d"}))
+		if tt.name == "file with empty content" {
```

Comment:
This condition is a bit buried. Couldn't the table-based tests indicate this condition more explicitly?

For example. We could add a field `responseStatus` to each test case and set it to 200 for most tests and to 422 for this one